libs/Microsoft.MixedReality.WebRTC.Native/include/interop_api.h

```diff
@@ -453,6 +453,10 @@ MRS_API void MRS_CALL mrsPeerConnectionRegisterRemoteAudioFrameCallback(
     PeerConnectionAudioFrameCallback callback,
     void* user_data) noexcept;
 
+// Experimental. Play or not remote audio tracks on the audio device.
+MRS_API void MRS_CALL
+mrsPeerConnectionPlayRemoteAudio(PeerConnectionHandle peerHandle, bool play);
```

Comment:
Should we call this 'RenderRemoteAudio' (sending data to device), not 'Play' (possible confusion vs paused)?